cmd/export.go

```diff
@@ -117,3 +135,36 @@ func promptDashboardIDs(kibanaClient *kibana.Client) ([]string, error) {
 	}
 	return selected, nil
 }
+
+func exportInstalledObjectsCmd(cmd *cobra.Command, args []string) error {
+	packageName, err := cmd.Flags().GetString(cobraext.ExportPackageFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.ExportPackageFlagName)
+	}
+
+	outputPath, err := cmd.Flags().GetString(cobraext.ExportOutputFlagName)
+	if err != nil {
+		return cobraext.FlagParsingError(err, cobraext.ExportOutputFlagName)
+	}
+
+	tlsSkipVerify, _ := cmd.Flags().GetBool(cobraext.TLSSkipVerifyFlagName)
+
+	clientOptions := elasticsearch.DefaultClientOptionsFromEnv()
+	clientOptions.SkipTLSVerify = tlsSkipVerify
```

Comment:
I think that in Go I prefer to avoid method chaining for initialization, but I will give it a try here as it seems we are using this approach in other initializations.

Reply:
I guess it's a sort of a personal preference. I think it's more important to keep it consistent, but you can consider this comment as a nit-pick